app/models/miq_server/worker_management/process.rb

```diff
@@ -4,7 +4,7 @@ def sync_from_system
     self.miq_processes = Sys::ProcTable.ps.select { |proc| proc.ppid == my_server.pid }
   end
 
-  def sync_starting_workers!(_starting)
+  def sync_starting_workers
   end
```

Comment:
Now that we're only returning the list of starting workers from this method I think we have to do `MiqWorker.find_all_starting.to_a` here otherwise the process mode will fail